src/azul/plugins/metadata/hca/transform.py

```diff
@@ -543,6 +546,19 @@ def _publication(self, p: api.ProjectPublication):
             "publication_url": p.publication_url
         }
 
+    @classmethod
+    def _accession_types(cls) -> FieldTypes:
+        return {
+            'domain': null_str,
+            'value': null_str
```

Comment:
Based on Slack discussion

https://humancellatlas.slack.com/archives/C01360XN04S/p1630428305017100

```suggestion
            'namespace': null_str,
            'accession': null_str
```

Upstream and downstream. This requires a change to the Metadata API. The corresponding PR should be attached to this ticket.

Reply:
opened https://github.com/DataBiosphere/hca-metadata-api/pull/58